tests/test_qcschema.py

```diff
@@ -0,0 +1,229 @@
+# This file is part of xtb.
+#
+# Copyright (C) 2020 Sebastian Ehlert
+#
+# xtb is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# xtb is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with xtb.  If not, see <https://www.gnu.org/licenses/>.
+
+from pytest import approx
+from xtb.qcschema.harness import run_qcschema
+import qcelemental as qcel
+import numpy as np
+
+def test_gfn2xtb_energy():
+    """Use QCSchema to calculate the energy a halogen bond compoind"""
+    thr = 1.0e-8
+
+    atomic_input = qcel.models.AtomicInput(
+        molecule = qcel.models.Molecule(
```

Comment:
can set molecule to a dict. pydantic should do the recursive validation automatically.